Update Best Practices Guide (#3108)
owncloud/docs: docs/best-practices.md

....
```

## Attributes

Reference: [`Document Attributes`](https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes/)
Reference: [`Inline Passthroughs`](https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/)

> Attributes are a way of creating a variable with a content which then can be used throuout the document. Using attributes increases

Author: ```suggestion
Attributes are a way of creating a variable with a content which then can be used throughout the document. Using attributes increases
```

> Attributes are a way of creating a variable with a content which then can be used throughout the document. Using attributes increases

Reviewer: ```suggestion
Attributes are a way of creating a variable with content which then can be used throughout the document. Using attributes increases
```

readability a lot and makes it easy to change content one time for all locations used. Attributes are written on top of the page

> and have folloing form:

Reviewer: ```suggestion
and have the following form:
```

Definition:
```
:name-of-an-attribute: value
```

Usage:
```
This is some text {name-of-an-attribute} which continues here.
```

> Attributes can also be used in scripts when you want to handover common used values. In case, you must define that attributes are

Author: ```suggestion
Attributes can also be used in scripts when you want to handover common used values. In case, you must define that attributes and macros
```

> Attributes can also be used in scripts when you want to handover common used values. In case, you must define that attributes and macros

Reviewer: ```suggestion
Attributes can also be used in scripts when you want to handover common used values. In this case, you must define the attributes and macros that
```

> handed over to the script for processing:

Author: ```suggestion
are handed over to the script for processing. "macros" is eg. needed, when using inline passthroughs:
```

```

> [subs=attributes+]

Author: ```suggestion
[subs=attributes+,+macros]
```

----

> incude...

Reviewer: ```suggestion
include...
```

|===
```

## TabSets

> ownCloud has added the Antora tabset extension for the documentation. With tabsets, you can create tabs inside

Author: ```suggestion
ownCloud has added the AsciiDoc tabset extension for the documentation. With tabsets, you can create tabs inside
```

> your document which is very useful for example scripts in different languages for the same task.

Reviewer: ```suggestion
your document which is very useful, for example, for scripts in different languages for the same task.
```
